cypress/spec/create_work.cy.js

```diff
@@ -0,0 +1,57 @@
+describe('Create work', () => {
+    let work_id
+
+    beforeEach(() => {
+      const results = cy.appFactories([
+        ['create', 'work_version_with_work_and_collection', 'with_required_associations', {} ]
+      ]).then((results) => {
+        cy.log(results[0])
+        work_id = results[0].work_id
+
+        cy.visit(`/works/${work_id}/edit`)
+    })
+    })
+
+    it('deposits a work correctly after uploading a file', () => {
+
+      // try to deposit
+      cy.get('input.btn[value="Deposit"]').click()
+
+      // there is a message telling us we need to upload a file
+      cy.get('div.invalid-feedback').should('contain', 'You must attach a file')
+
+      // now upload a file
+      cy.get('div.dropzone').selectFile('cypress/fixtures/test.txt', {
+        action: 'drag-drop'
+      })
+
+      // wait for the upload to finish
+      cy.wait(1000)
```

Comment:
Is there something you can wait for to appear rather than just a wait?

Reply:
I tried waiting for the filename and checkmark to appear after uploading, and while those assertions do work, the test will still always fail.  It looks like the click of the "Deposit" button doesn't work unless you pause for just a bit after uploading the file.  I'm not sure why, perhaps some interaction with the file upload component.